packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.ts

```diff
@@ -566,8 +566,8 @@ const rule = {
           // Is React managing this ref or us?
           // Let's see if we can find a .current assignment.
           let foundCurrentAssignment = false;
-          for (const reference of references) {
-            const {identifier} = reference;
+          for (const ref of references) {
```

Comment:
There was already a `reference` in scope, and eslint was complaining about it after hooking up the new lint config.